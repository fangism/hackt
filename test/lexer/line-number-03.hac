/*	"line-number-03.hac"						*/
/*	old bug: multi-line C-comments caused line-number count		*/
/*	to be off-by one for each block.				*/
/*	contributed by Clint Kelly, further reduced by Fang		*/
/******* ********* ******* ******** ******** ******* ***** ***** ****** */

/** single-line block comments however did not have this problem **/

defproc e1of1 (bool e, d) { }

defproc test (bool _sR, _pR) {

e1of1 l, x, r;
bool _l, _x, _r;
bool xv, rv;

prs {
	l.e => _l-
	_l => l.d-

	x.e & l -> _x-		// error is here, line 21
}
}


