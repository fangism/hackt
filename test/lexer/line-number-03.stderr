ERROR: expression at [21:8] does not reference a bool.
ERROR building PRS-expr at [21:8]
ERROR: at least one error in PRS body.
